src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int MAX_RANDOM = 100;
+    public static final int DEFAULT_NUMBER = 10;
+    public static final String DEFAULT_COLOR = "white";
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int index = random.nextInt(FIGURE_COUNT);
+
+        switch (index) {
+            case 0:
+                return getRandomSquare();
+            case 1:
+                return getRandomRectangle();
+            case 2:
+                return getRandomRightTriangle();
+            case 3:
+                return getRandomCircle();
+            case 4:
+                return getRandomIsoscelesTrapezoid();
+            default:
+                return getDefaultFigure();
+        }
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle(DEFAULT_NUMBER, DEFAULT_COLOR);
+    }
+
+    public Figure getRandomSquare() {
```

Comment:
FigureSupplier should have only two public methods(getRandomFigure, getDefaultFigure)
```suggestion
    private Figure getRandomSquare() {
```